app/src/main/java/org/wikipedia/editactionfeed/EditTasksFragment.java

```diff
@@ -137,35 +140,74 @@ public void onNegativeActionClick() {
 
             @Override
             public void onViewClick() {
-                startActivity(AddTitleDescriptionsActivity.Companion.newIntent(requireActivity(), EDIT_TASKS_TITLE_DESC_SOURCE));
+                startActivity(AddTitleDescriptionsActivity.Companion.newIntent(requireActivity(), InvokeSource.EDIT_FEED_TITLE_DESC));
             }
         });
 
-        EditTask multilingualTask = new EditTask();
-        multilingualTask.setTitle(getString(R.string.translation_task_title));
-        multilingualTask.setDescription(getString(R.string.translation_task_description));
-        multilingualTask.setImagePlaceHolderShown(true);
-        multilingualTask.setImageDrawable(ContextCompat.getDrawable(requireContext(), R.drawable.ic_icon_translate_title_descriptions));
-        multilingualTask.setNoActionLayout(Prefs.isEditActionTranslateDescriptionsUnlocked());
-        multilingualTask.setDisabled(!Prefs.isEditActionTranslateDescriptionsUnlocked());
-        multilingualTask.setDisabledDescriptionText(String.format(getString(R.string.translate_description_edit_disable_text), 2));
-        tasks.add(multilingualTask);
-        callbacks.add(new EditTaskView.Callback() {
-            @Override
-            public void onPositiveActionClick() {
-            }
+        if (WikipediaApp.getInstance().language().getAppLanguageCodes().size() == 1) {
+            EditTask multilingualTask = new EditTask();
+            multilingualTask.setTitle(getString(R.string.multilingual_task_title));
```

Comment:
Naming is confusing...
We have `multilingualTask` here (line 148), and also another `multilingualTask` on line 185.

Reply:
yes... they are the same translation task but with different looking UIs for since and multiple secondary language(s) users respectively.. Should i name it differently for readability... they are in different code boundaries so logic-wise we are fine..